src/main/java/com/icthh/xm/ms/balance/service/mapper/BalanceChangeEventMapper.java

```diff
@@ -8,13 +8,19 @@
 /**
  * Mapper for the entity BalanceChangeEvent and its DTO BalanceChangeEventDto.
  */
-@Mapper(componentModel = "spring", uses = {})
+@Mapper(componentModel = "spring")
 public interface BalanceChangeEventMapper extends EntityMapper<BalanceChangeEventDto, BalanceChangeEvent> {
     @Override
     @Mapping(target = "metadata", source = "metadata.metadata" )
     BalanceChangeEventDto toDto(BalanceChangeEvent entity);
 
-    @Override
     @Mapping(target = "metadata.metadata", source = "metadata" )
+    @Mapping(target = "id", ignore = true)
+    @Mapping(target = "pocketChangeEvents", ignore = true)
+    @Mapping(target = "executedByUserKey", ignore = true)
+    @Mapping(target = "balanceTypeKey", ignore = true)
+    @Mapping(target = "balanceKey", ignore = true)
+    @Mapping(target = "balanceEntityId", ignore = true)
```

Comment:
please explain why this new annotations is needed

Reply:
Added this annotations to avoid mapstruct warnings during compilation.
MapStruct says that those field will be ignored during mapping. These annotations doesn't change the behavior of the application, but disable MapStruct mapping warnings.